global.go

```diff
@@ -0,0 +1,30 @@
+package netdicom
+
+var allTransferSyntaxes = []string{
```

Comment:
Also, please change `allTransferSyntaxes` type into `map[string]struct{}` for more efficient lookup.

Reply:
Go maps do not maintain the insertion order (https://go.dev/ref/spec#Map_types), list is more fit in this case